lib/components/achievement_dialog.dart

```diff
@@ -10,8 +10,8 @@ import 'package:share_plus/share_plus.dart';
 
 import '../graphql/queries/achievements.graphql.dart';
 import '../helpers/ui/btv_buttons.dart';
-import '../theme/bccm_colors.dart';
-import '../theme/bccm_typography.dart';
+import '../theme/design_system/design_system.dart';
+import '../theme/design_system/design_system.dart';
```

Comment:
Just noticed that there are a lot of files importing `design_system.dart` twice.